aries_cloudagent/protocols/present_proof/v1_0/manager.py

```diff
@@ -324,10 +324,13 @@ async def receive_presentation(
         # But present proof supports the old-style AIP-1 connectionless exchange that
         # bypasses the oob record. So we can't verify if an oob record is associated with
         # the exchange because it is possible that there is None
+        #
+        # A connectionless proof doesn't have a connection_id, so default to None
+        # even if there is no oob record.
         connection_id = (
             None
             if oob_record
-            else connection_record.connection_id
+            else getattr(connection_record, 'connection_id', None)
```

Comment:
I'm not a fan of using `getattr` if it can be avoided. On top of that, the double layered ternary is difficult to parse correctly. I think I'd recommend breaking this up into more lines to make it clear where the connection_id is being set and under what circumstances.

Reply:
Got rid of getattr and (hopefully) made things more readable.